electron/main.ts

```diff
@@ -597,9 +597,13 @@ ipcMain.handle(
 
 /// IPC handlers begin here.
 
-ipcMain.handle('checkGameUpdates', async () => {
-  const legendaryUpdates = await LegendaryLibrary.get().listUpdateableGames()
-  const gogUpdates = await GOGLibrary.get().listUpdateableGames()
+ipcMain.handle('checkGameUpdates', async (event, libraries) => {
```

Comment:
I believe you should receive a `Runner` here and then depending on the Runner you call the right one.
Use a `switch` to make it more scalable. :)

Reply:
could also have a full parameter to refresh all libraries